Sources/BraintreeCard/BTCard.m

```diff
@@ -151,13 +95,8 @@ - (NSDictionary *)parameters {
     if (billingAddressDictionary.count > 0) {
         p[@"billing_address"] = [billingAddressDictionary copy];
     }
-    
-    NSMutableDictionary *optionsDictionary = [NSMutableDictionary new];
-    if ([p[@"options"] isKindOfClass:[NSDictionary class]]) {
-        [optionsDictionary addEntriesFromDictionary:p[@"options"]];
-    }
```

Comment:
Similar to my comment above about billing address, this was coping over any options fields passed in through the `initWithParameters` initializer, which we're removing.